docs/end_to_end_tutorials/dev_practices/e2e_evaluation.md

```diff
@@ -0,0 +1,45 @@
+# End-to-End Evaluation
```

Comment:
can we define what "end to end evaluation" is and how it is different from component wise evaluation, and why you would still want to test it

Reply:
As mentioned above, I view e2e eval as the gold standard and the rest as optional